lib/libc/arcmwdt/include/errno.h

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 1984-1999, 2012 Wind River Systems, Inc.
+ * Copyright (c) 2023 Synopsys
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief System error numbers
+ */
+
+#ifndef LIB_LIBC_ARCMWDT_INCLUDE_ERRNO_H_
+#define LIB_LIBC_ARCMWDT_INCLUDE_ERRNO_H_
+
+#include <zephyr/sys/errno_private.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include_next <errno.h>
```

Comment:
Should this include be before or after the ' extern "C" ' part? For limit.h it is after, here it is before. Let's be consistent, it is probably cleaner to have it outside the cplusplus block.

Reply:
Fixed